libraries/mbed/targets/hal/TARGET_NXP/TARGET_LPC15XX/analogin_api.c

```diff
@@ -90,8 +86,8 @@ void analogin_init(analogin_t *obj, PinName pin) {
     adc_reg->CTRL = (1UL << 30) | (clkdiv & 0xFF);
     while ((adc_reg->CTRL & (1UL << 30)) != 0);
 
-    // switch to asynchronous mode
-    adc_reg->CTRL = (1UL << 8);
+    // Sampling clock: SystemClock divided by 1
+    adc_reg->CTRL = 0;
```

Comment:
NOTE: Running the ADC at 72MHz _(SystemCoreClock / 1)_ didn't appear to affect the readings on my test board. If issues arise in the future, _SystemCoreClock_ can simply be scaled using the CLKDIV bits in the CTRL register.